api/envoy/service/load_stats/v2alpha/BUILD

```diff
@@ -0,0 +1,14 @@
+load("@envoy_api//bazel:api_build_system.bzl", "api_go_grpc_library", "api_go_proto_library", "api_proto_library_internal")
+
+licenses(["notice"])  # Apache 2
+
+api_proto_library_internal(
+    name = "orca",
+    srcs = ["orca.proto"],
+    has_services = 1,
+)
+
+api_go_grpc_library(
+    name = "orca",
+    proto = ":orca",
+)
```

Comment:
@mattklein123 might this make sense as our first UDPA API? I'm thinking maybe it could live in `udpa/data/v1` for the message and `udpa/service/orca/v1` for the service definition? This is a data model used for backend load reporting, and will have various realizations (JSON, YAML, HTTP header).

Reply:
Sure let's do it. I can take a look at the change once the file has been moved.

Reply:
@htuch 
I would think about putting `OrcaLoadRepoprt` message under `api/envoy/data/orca/v1` and `OrcaLoadReportRequest` message + `OpenRcaService` in a single proto under `api/envoy/service/orca/v1`. What do you think about?